### DISCUSSION TITLE
Blazor Component Library incorporating BeerCSS

### DISCUSSION THREAD

**Discussion by Kristianfriis** [beercss/beercss#174] [Q&A] [answered]
Hi, 

First of all, love BeerCSS. Using it for a home project where we can track what we have in the freezers. 
The design is cool, and it is easy to implement the various components. 

I was just wondering if there was any movement towards building a Blazor UI Component Library on top of BeerCSS? I myself is using Blazor and have made some wrapper components, but could be cool if something "official" could be obtained. 

If you are interested I can share my bare boned wrapper library :-)

**Comment by KicKerBNU** [comment, marked as answer]
Hello @Kristianfriis, 

Love to hear that you are liking work with Beercss especially with C#  instead of Javascript, its always helpfull to see diferent projects and approachs using Beercss if you can share your repository could help other developers who want to use the same strategy of development.

Right now we dont have any official documentation about the use of Beercss with Blazor maybe with your help we can create a good starting point.

Cheers
🍺

**Reply by Kristianfriis** [reply]
Hi @KicKerBNU 

Will do 😀 I have made a new show and tell entry 😃
